lib/utils/device/deploy.ts

```diff
@@ -140,9 +141,11 @@ export async function deployToDevice(opts: DeviceDeployOptions): Promise<void> {
 		globalLogger.logDebug('Checking we can access device');
 		await api.ping();
 	} catch (e) {
-		throw new ExpectedError(
-			`Could not communicate with local mode device at address ${opts.deviceHost}`,
-		);
+		throw new ExpectedError(stripIndent`
+			Could not communicate with local mode device at address ${opts.deviceHost}.
```

Comment:
In this sentence, `local mode device` implies the device is known to be in local mode, which is then readily cast into doubt by the subsequent sentence. :-) Anyway, I think it would be nice to be specific that the CLI was not able to contact the supervisor on the device:
```
	Could not communicate with device supervisor at address ${opts.deviceHost}.
	Device may not have local mode enabled. Check with:
	  balena device local-mode <device-uuid>
```
We could even mention the port number. A few lines above (line 137), we could do:
```
const port = 48484;
const api = new DeviceAPI(globalLogger, opts.deviceHost, port);
```
And then:
```
	Could not communicate with device supervisor at address ${opts.deviceHost}:${port}.
	Device may not have local mode enabled. Check with:
	  balena device local-mode <device-uuid>
```